.gitlab-ci.yml

```diff
@@ -1250,38 +1289,64 @@ deploy_process_and_sysprobe:
 # Docker releases
 #
 
-tag_release:
+tag_release_6:
   <<: *docker_tag_job_definition
   <<: *run_when_triggered_on_tag
   stage: deploy
   when: manual
   variables:
     <<: *docker_hub_variables
   script:
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG} datadog/agent:${CI_COMMIT_TAG}
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-jmx datadog/agent:${CI_COMMIT_TAG}-jmx
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2 datadog/agent:${CI_COMMIT_TAG}
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py2-jmx datadog/agent:${CI_COMMIT_TAG}-jmx
     - inv -e docker.publish --signed-push ${SRC_DSD}:${SRC_TAG} datadog/dogstatsd:${CI_COMMIT_TAG}
 
-latest_release:
+tag_release_7:
   <<: *docker_tag_job_definition
   <<: *run_when_triggered_on_tag
   stage: deploy
   when: manual
   variables:
     <<: *docker_hub_variables
   script:
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG} datadog/agent:latest
-    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-jmx datadog/agent:latest-jmx
+    - inv -e docker.publish --signed-push ${SRC_AGENT}:${SRC_TAG}-py3 datadog/agent:${CI_COMMIT_TAG}
```

Comment:
Shouldn't we push with the `-py3` prefix here ?

Reply:
The idea there is that python 2 and python 3 releases live under different tags (and thus commits). So if we want to draft a beta python 3 packages we just need to tag 6.X-beta-py3-1 for instance as the commit tag and images will be pushed as datadog/agent:6.X-beta-py3-1 and datadog/agent:6.X-beta-py3-1-jmx. Then later with agent 7 it will be datadog/agent:7.X and datadog/agent:7.X-jmx.

Reply:
Sounds good ! Could you update the release doc once this is merged ?

Reply:
Sure